zmon-controller-app/src/test/java/org/zalando/zmon/controller/MultiKairosDBControllerTest.java

```diff
@@ -63,7 +64,7 @@ public void setUp() throws MalformedURLException {
 
         this.mockMvc = MockMvcBuilders
                 .standaloneSetup(new MultiKairosDBController(properties, metricsRegistry,
-                        new AsyncRestTemplate(new HttpComponentsAsyncClientHttpRequestFactory()), accessTokens))
+                        new AsyncRestTemplate(new HttpComponentsAsyncClientHttpRequestFactory()), accessTokens, GlobalTracer.get()))
```

Comment:
It seems to me that we could inject a test tracer that would allow you to test that the span tags were actually inserted according to your expectations. This would effectively test the new functionality which, currently, is not tested at all.